src/site/content/en/blog/mailru-cwv/index.md

```diff
@@ -0,0 +1,290 @@
+---
+layout: post
+title: Improving Core Web Vitals on the Mail.ru home page resulted in an average 10% increase in conversion rates
+subhead: |
+  Several months of work to improve Core Web Vitals (CWV) metrics on the home page of Mail.ru resulted in a 60% increase in the 75th percentile in Cumulative Layout Shift (CLS), boosting average session time by 2.7% and conversion rates of core sections by more than 10%.
+description: |
+  Several months of work to improve Core Web Vitals (CWV) metrics on the home page of Mail.ru resulted in a 60% increase in the 75th percentile in Cumulative Layout Shift (CLS), boosting average session time by 2.7% and conversion rates of core sections by more than 10%.
+authors:
+  - dstasyev
+  - svenmay
+date: 2021-11-15
+hero: image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/D3aV6VLiccC31BJIwo3z.jpg
+alt: Google Core Web Vitals + Mail.ru Home Page.
+thumbnail: image/vgdbNJBYHma2o62ZqYmcnkq3j0o1/CWMTBHzJ9UCpZ89fNcxJ.jpg
+tags:
+  - blog
+  - web-vitals
+  - case-study
+---
+
+{% Aside %}
+  This case study was authored by Denis Stasyev from the Mail.ru infrastructure team.
+{% endAside %}
```

Comment:
```suggestion
```

Reply:
I'm not sure we need this. Denis appears to be on the byline. If you're concerned that having you (Sven, I mean) as an author is an issue for any reason without this context, a good approach is to always throw in a final paragraph (usually italicized) that gives a special thanks/shoutout to other editors/contributors/etc.